### PR TITLE
Update Adlibris.xml

### PR DESCRIPTION
**Pull request by pipboy96**
No evidence of mixed content issues.

### CODE REVIEW

File: src/chrome/content/rules/Adlibris.xml [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-<ruleset name="Adlibris/Capris" platform="mixedcontent" default_off='failed ruleset test'>
-  <target host="www.adlibris.com" />
-  <target host="adlibris.com" />
-  <target host="www.capris.no" />
-  <target host="capris.no" />
+	<securecookie host=".+" name=".+" />
 
-  <rule from="^http://(?:www\.)?(adlibris\.com|capris\.no)/"
-          to="https://www.$1/"/>
+	<rule from="^http://(www\.)?capris\.no/" to="https://www.adlibris.com/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Adlibris.xml]: Please add a comment for it.

Comment (pipboy96) [reply src/chrome/content/rules/Adlibris.xml]: This doesn't seem to be required anymore.

Comment (J0WI) [reply src/chrome/content/rules/Adlibris.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/497516138#L569-L570

Comment (pipboy96) [reply src/chrome/content/rules/Adlibris.xml]: For some inexplicable reason I can access both without problem.

Comment (pipboy96) [reply src/chrome/content/rules/Adlibris.xml]: Done.

**Pull request merged**